src/Microsoft.ML.Data/Transforms/ConcatTransform.cs

```diff
@@ -536,10 +536,11 @@ private static VersionInfo GetVersionInfo()
         public override ISchema Schema => _bindings;
 
         /// <summary>
-        /// Convenience constructor for public facing API.
+        /// Convenience constructor for creating <see cref="ConcatTransform"/>.
+        /// The <see cref="ConcatTransform"/> concatenates two or more columns (either scalar or vector) of same type into a single vector column.
         /// </summary>
         /// <param name="env">Host Environment.</param>
-        /// <param name="input">Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
```

Comment:
>his is the output from previous transform or loade [](start = 64, length = 50)

why is this going away?

Reply:
I added this but @TomFinley had a comment in previous PR to remove this part. Let see other reviewers' feedback on this.

Reply:
I added this but @TomFinley had a comment in previous PR to remove this part. Let see other reviewers' feedback on this.

---
In reply to: [207687012](https://github.com/dotnet/machinelearning/pull/646#discussion_r207687012) [](ancestors = 207687012)